server/service/src/sync/test/integration/omsupply_central/vaccine_card.rs

```diff
@@ -0,0 +1,228 @@
+use super::sync_omsupply_central;
+use crate::{
+    programs::patient::{link_patient_to_store, patient_updated::create_patient_name_store_join},
+    sync::{
+        test::{
+            check_integrated,
+            integration::{
+                central_server_configurations::ConfigureCentralServer, create_site,
+                init_test_context, integrate_with_is_sync_reset, GraphqlRequest,
+            },
+        },
+        translations::IntegrationOperation,
+        CentralServerConfig,
+    },
+};
+use repository::{GenderType, NameRow, NameRowType, VaccinationRow};
+use serde_json::json;
+use util::uuid::uuid;
+
+use super::graphql;
+
+static ADD_VACCINE_COURSE: &str = r#"
+mutation MyMutation(
```

Comment:
in this test you will find, i used only the minimum, which could break in the future, but I think it's ok for now